src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -307,6 +307,7 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
     val fileContent =
       s"""
          |#!/bin/bash
+         |export _JAVA_OPTIONS=-Djava.io.tmpdir=$JesCromwellRoot/tmp
```

Comment:
What about also `export TMPDIR=$JesCromwellRoot/tmp` so languages that honor the `TMPDIR` environment variable won't break too?